### DISCUSSION TITLE
ANPHON 1.4.2 can't calculate Bubble self-energy

### DISCUSSION THREAD

**Discussion by Lamed-git** [ttadano/alamode#140] [Q&A]
Dear Dr. Tadano,
I read your previous article https://doi.org/10.7566/JPSJ.87.041015, which mentioned the correction of cubic anharmonicity for phonon spectrum under finite temperature -bubble self energy, but I found ANPHON V1.4.2 did not mention similar calculation. I have read the previous discussion and found that the setting of SELF_W=1 can work, but ANPHON 1.4.2 does not recognize this parameter. I wonder whether this function has been removed. If I still want to use this function, which version of ANPHON should I use?
Looking forward to your reply!
Thanks,
Lamed

**Comment by ZhunyunTang99** [comment]
Have you solved this problem? I also want to know how to add the bubble term to the frequency correction of anharmonic phonon.

Best regards
Tang

**Reply by Lamed-git** [reply]
You can use consider using an earlier version, such as 1.4.1. Please refer to https://github.com/ttadano/alamode/discussions/38 this discussion for guidance on calculating bubble energy.

**Reply by gangliu-github** [reply]
Can 1.5.0 calculate the bubble self energy？

**Reply by Lamed-git** [reply]
You can try it by add SELF_W=1.

**Reply by AinDin64** [reply]
Dear teacher，I have a question for calculating the bubble free energy. The version 1.5.0 can not recognize the variable FE_BUBBLE, do you known how to calculate the bubble free energy?
